### PR TITLE
aarch64: Use latest released rhel9 kernel

### PR DESCRIPTION
**Pull request by cfergeau**
swap_kernel is downloading kernel packages directly from
http://download.eng.bos.redhat.com/rhel-9/rel-eng/RHEL-9/latest-RHEL-9/
This repo contains a frozen set of packages from the time RHEL9 was released.
What we really want is the latest version which was released through
zstream.

This commit switches to a different URL to fetch the packages which
should be more representative to the current state of rhel9. It also
makes use of a .repo file in order to reuse yum to fetch the needed files.

### CODE REVIEW

**Comment by praveenkumar** [comment]
@cfergeau I will give this a try tomorrow since our CI only check x86 changes

**Comment by cfergeau** [comment]
> @cfergeau I will give this a try tomorrow since our CI only check x86 changes

Sure, I've been able to successfully run 2 iterations of this PR after some initial unrelated issues.

**Comment by praveenkumar** [comment]
> > @cfergeau I will give this a try tomorrow since our CI only check x86 changes
> 
> Sure, I've been able to successfully run 2 iterations of this PR after some initial unrelated issues.

@cfergeau In that case I am going to merge it.

File: createdisk-library.sh [hunk 1]
```diff
@@ -177,6 +177,19 @@ function install_additional_packages() {
     fi
 }
 
+# This is only needed for creating arm64 bundles until RHCOS switches to RHEL-9
+function install_rhel9_kernel {
+    local vm_ip=$1
+    local pkgDir=$(mktemp -d tmp-rpmXXX)
+
+    mkdir -p ${pkgDir}/packages
+    yum download --setopt=reposdir=./repos --setopt=sslcacert=./repos/2015-RH-IT-Root-CA.pem --downloadonly --downloaddir ${pkgDir}/packages kernel kernel-modules-extra --resolve
```
Comment (praveenkumar) [inline createdisk-library.sh:186]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**